broker/broker-undelivered-messages-ui/src/main/resources/WEB-INF/web.xml

```diff
@@ -40,5 +40,20 @@
             <transport-guarantee>CONFIDENTIAL</transport-guarantee>
         </user-data-constraint>
     </security-constraint>
+
+    <filter>
+        <filter-name>GzipFilter</filter-name>
+        <filter-class>org.eclipse.jetty.servlets.GzipFilter</filter-class>
+        <init-param>
+            <param-name>mimeTypes</param-name>
+
+            <param-value>text/html,text/plain,text/xml,application/xhtml+xml,text/css,application/javascript,image/svg+xml</param-value>
```

Comment:
Are these the only mime types that are run through gzip? Have you tried looking at `application/json`?